Tank.java

```diff
@@ -3,23 +3,28 @@
 import java.util.*;
 
 public class Tank {
-	public static  int speedX = 6, speedY =6; 
+	public static int speedX = 6, speedY =6;
 	public static int count = 0;

```

Comment:
"public static" fields should be constant
Ar yra priežastis lauką paskelbti „public“ ir „static“, neskelbiant jo „final“?